tests/NHSD.BuyingCatalogue.Solutions.Application.UnitTests/Solutions/Suppliers/GetSuppliersByNameTests.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Moq;
+using NHSD.BuyingCatalogue.Solutions.Contracts.Persistence;
+using NHSD.BuyingCatalogue.Solutions.Contracts.Queries;
+using NUnit.Framework;
+
+namespace NHSD.BuyingCatalogue.Solutions.Application.UnitTests.Solutions.Suppliers
+{
+    [TestFixture]
+    internal sealed class GetSuppliersByNameTests
```

Comment:
Is this to test the `GetSuppliersByNameHandler`?

Reply:
Actually ignore this. It looks like only one "Handler" test class has the name "Handler" in it. Took me slightly longer to figure out what the System Under Test (SUT) was when the name of the test does not start with the full class name.

Reply:
I found that slightly confusing at first but I decided to follow what looked like the existing pattern – possibly some room for improvement here across the handler tests as tech debt...